ext/nnstreamer/tensor_decoder/nnstreamer.proto

```diff
@@ -0,0 +1,34 @@
+syntax = "proto3";
+
+package NNStreamer;
+
+message Tensor {
+  string name = 1;
+  enum Tensor_type {
+    NNS_INT32 = 0;
+    NNS_UINT32 = 1;
+    NNS_INT16 = 2;
+    NNS_UINT16 = 3;
+    NNS_INT8 = 4;
+    NNS_UINT8 = 5;
+    NNS_FLOAT64 = 6;
+    NNS_FLOAT32 = 7;
+    NNS_INT64 = 8;
+    NNS_UINT64 = 9;
+  
+    NNS_END = 10;
+  }
+  Tensor_type type = 2;
+  string dimension = 3;
```

Comment:
Why declare dimension as string?

Reply:
You're right. I changed it to `repeated uint32 dimension` .